atlp-rwanda/atlp-pulse-mobile
# #10 Account password reset

### What does this PR do?

This PR includes designs and implementations of trainee password reset.

### Description of Task to be completed?

If a user wants to change their password, they provide the current password and a pair of new password. If the current password is correct the password is updated otherwise an appropriate error message is thrown.

### How should this be manually tested?

- Open the app and login.
- Click on the profile picture and navigate to account tab
- Fill in the form and logout
- Check if the previous password is working and if the new password is working

### Screenshots

<img width="300" src="https://github.com/user-attachments/assets/e3c0df6b-e9fd-41bb-a4a0-8966bedd8e16" />
<img width="300" src="https://github.com/user-attachments/assets/38d25f58-28e5-4ce8-b155-c2ba3356572e" />
